packages/reactive-element/src/reactive-element.ts

```diff
@@ -781,6 +790,11 @@ export abstract class ReactiveElement
     if (properties === undefined) {
       return;
     }
+    // Grab reactive properties from superclass that may have been added by
+    // metadata.
+    for (const [k, v] of superCtor.elementProperties.entries()) {
+      this.elementProperties.set(k, v);
+    }
```

Comment:
The timing of this bothers me. If I lift this up to L784 it results in other tests failing. But here or on L789 is the perfect spot – and I don't yet understand why.

Reply:
Nvm – it's a bug in this implementation. You can overwrite properties by lifting twice.

I think the simpler solution may be much more robust:

```
FinalizeBaseClass
CollectMetadata BaseClass
FinalizeSubClass
CollectMetadata SubClass
```

But adding a bunch more tests.